fe/plugin-common/src/main/java/com/starrocks/common/io/BitmapValue.java

```diff
@@ -15,7 +15,7 @@
 // specific language governing permissions and limitations
 // under the License.
 
-package com.starrocks.load.loadv2.dpp;
+package com.starrocks.common.io;
```

Comment:
Is it appropriate to put bitmap value in common.io?

Reply:
Is it OK to put it in com.starrocks.types package?

Reply:
`types` is better, fixed